templates/ts-apollo-fullstack/.graphqlrc.yml

```diff
@@ -1,40 +1,27 @@
 ## GraphQL Config Generated by Graphback
 ## Please review configuration and adjust it for your own project
-schema: ./server/src/schema/*.graphql
```

Comment:
I think this file was commited by mistake (should be reverted?)

Reply:
Good catch.

Reply:
I noticed a couple of other `templates/apollo...` generated files , should they be ignored?